src/renderer/components/about.vue

```diff
@@ -1,33 +1,61 @@
 <template>
-    <div id="about" class="grid-container">
-        <div class="grid-x">
-            <div class="cell">
-                <h4>Contact us</h4>
-                <pre>This software is the result of several years of development, testing, doing and redoing.
-A lot of thought and effort went into making it work well.
-However, if you encounter any problems at your competition,
-or you have any suggestions or questions, please contact us:
-</pre>
-                <div class="grid-x text-center grid-padding-y">
-                    <div class="cell small-4">Email FLL-TMS-Support@firstinspires.org</div>
-                    <div class="cell small-4"><a @click="event => openSite(event, 'http://fll-tools.com/support/')">Visit our website Support Page</a></div>
+    <div id="about" class="ui container">
+        <h1>Contact us</h1>
+        <p>
+            This software is the result of several years of development, testing, doing and redoing.
+            A lot of thought and effort went into making it work well.
+            However, if you encounter any problems at your competition,
+            or you have any suggestions or questions, you can get support:
+        </p>
+        <div class="ui three cards">
```

Comment:
Great usage of cards! Please put them in a `padded grid > row` so they would have a margin from them top and bottom.

Reply:
I ddin't use `padded grid` but I fix the problem